### ISSUE TITLE
[Visual requirement - Make code - History]: "History", "Release Zone" and "Extension Zone" texts appear as heading but not defined under any heading.

### ISSUE THREAD

**Issue by Yamini234u** [microsoft/pxt-microbit#5503]
### **User Experience**
Visually impaired people who depend on screen readers for navigation will be impacted If any text visually appears as heading is not defined programmatically makes it difficult for the end-users to navigate the dialogs/page in heading mode as they will miss the important section while navigating the screen.

**Note:**
User credentials should NOT be included in the bug.

### **Repro Steps:**
1. Launch application URL : https://makecode.microbit.org
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate to all the controls under History, Release Zone, Extension Zone Section.
7. Verify whether 'History, Release Zone, Extension Zone ' text is visually appears like heading but programmatically defined or not.
### **Actual Result:**
"History", "Release Zone" and "Extension Zone" texts appear as headings but are not defined under any heading tag.
**Observation:** "History", "Release Zone" and "Extension Zone" texts are defined "div" tag.
### **Expected Result:**
 "History", "Release Zone" and "Extension Zone" texts should be defined under "H3" tag.
### **MAS Reference:**
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA
### **DOM details available:**
Yes(Please refer to the attachment)

### **Reference Links:**
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDc3NDY0MzExZDkxNTRlYmRhNzk2YTExYzE5Y2I2N2ZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3ODcwNDg4MTQ1NjM2NzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=VHBCSFBzMTdmWExVNTk3MzZlTHdiME4rcFo3d3lyYk5kTTIvRjhLdTN3QT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-chbhardwaj%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
### **Test Environment:**
- **OS Version**: Windows 11 23H2 (OS Build:22631.3007)
- **Browser:** Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- **Test Matrix:** Chrome + JAWS
- **URL:** https://makecode.microbit.org/.
<img width="965" alt="MAS1 3 1-History, Release Zone and Extension Zone texts appear as heading but not defined under any heading" src="https://github.com/microsoft/pxt-microbit/assets/117158108/1ff82497-116f-4760-889b-79c07eaf532a">

**Comment by Yamini234u** [comment]
#Accessibility;#A11yMAS;#A11ySev2;AILimited;#HCL;#E2E-HCL-MakeCode-Feb2024;#NonBenchmark;#Win11-Chrome;#MAS1.3.1
